docs/topics/logging.txt

```diff
@@ -241,25 +241,37 @@ information about logging configuration dictionaries. However, to give
 you a taste of what is possible, here are several examples.
 
 First, here's a simple configuration which writes all request logging from the
-:ref:`django-request-logger` logger to a local file::
+:ref:`django-logger` logger to console if ``DEBUG=True`` and sends email to admins if ``DEBUG=False``::
 
     LOGGING = {
         'version': 1,
         'disable_existing_loggers': False,
+        'filters': {
+            'require_debug_false': {
+                '()': 'django.utils.log.RequireDebugFalse',
+            },
+            'require_debug_true': {
+                '()': 'django.utils.log.RequireDebugTrue',
+            },
+        },
         'handlers': {
-            'file': {
-                'level': 'DEBUG',
-                'class': 'logging.FileHandler',
-                'filename': '/path/to/django/debug.log',
+            'console': {
+                'level': 'INFO',
+                'filters': ['require_debug_true'],
+                'class': 'logging.StreamHandler',
             },
+            'mail_admins': {
+                'level': 'ERROR',
+                'filters': ['require_debug_false'],
+                'class': 'django.utils.log.AdminEmailHandler'
+            }
         },
         'loggers': {
-            'django.request': {
-                'handlers': ['file'],
-                'level': 'DEBUG',
-                'propagate': True,
+            'django': {
+                'handlers': ['console', 'mail_admins'],
+                'level': 'INFO',
             },
-        },
```

Comment:
We prefer including a trailing comma so that if the dict is extended later, we don't have to modify that line again.